### ISSUE TITLE
/sponsors

### ISSUE THREAD

**Issue by hueniverse** [hapijs/hapijs.com#220]
Need a new page to list module sponsors. This will be linked to from the main header. The page should have two sections, current sponsors and alumni sponsors. It should feed off a JSON document where lead maintainers can PR changes. The logo images for this page should be stored in the site images directory under /sponsors prefix.

**Comment by danecando** [comment]
After adding sponsors to the header menu there's a break between 1201 & 1289px's before the mobile menu kicks in. Is there something specific you would like to do about this? IE: Scale down the logo, menu font size or spacing

![Uploading Screen Shot 2015-11-03 at 10.18.28 PM (2).png…]()

**Comment by franvarney** [comment]
@nlf and I discussed possibly making the menu two rows

**Comment by danecando** [comment]
That doesn't sound like a bad idea either! If you do decide on something let me know I'll be glad to make a PR. Already mostly done just figured I'd bring this up.

**Comment by nlf** [comment]
yeah i think the two row menu is the way to go for now, we just have too much going on in the menu to keep it at one row neatly now

**Comment by danecando** [comment]
More of a set two rows instead of them breaking into a second as it gets smaller? I did this example with some css without changing the markup to reflect two rows.

<img width="1440" alt="screen shot 2015-11-05 at 10 18 41 pm" src="https://cloud.githubusercontent.com/assets/3721977/10988591/a1af7d6a-840b-11e5-9178-d6a486374d46.png">

**Comment by hueniverse** [comment]
Maybe we have too many now and should move to a sidebar or sub menus?

**Comment by danecando** [comment]
There's a lot of room in the header section

<img width="1440" alt="screen shot 2015-11-05 at 10 42 58 pm" src="https://cloud.githubusercontent.com/assets/3721977/10988894/aa16d82e-840e-11e5-8cb1-84fa6ae5ea8b.png">

**Comment by AdriVanHoudt** [comment]
@danecando I like what you did there, maybe the top menu in black is not super clear but looks good nonetheless

**Comment by franvarney** [comment]
Agree with @AdriVanHoudt about the black. Doesn't look like something that is clickable. However, I think this is definitely the right direction!

**Comment by danecando** [comment]
*feelings hurt*

kidding of course.

Yeah was just tweaking around in dev tools to try to present some options. An extra sidebar menu could be nice. IE: Move the smaller nav with the blank links from my prev screenshot above or below the search box but then only a few pages actually have a sidebar and I don't think it would be great to shoehorn one in. 

I'll see what I can come up with later on tonight

**Comment by guahanweb** [comment]
Hey, guys, what's the status on this one? @danecando, did you end up doing anything with it? I was going to take a stab at putting the page itself together, but I didn't want to interfere with more global header/footer modifications.
